packages/core/src/elements/ResponsiveElement.ts

```diff
@@ -31,14 +31,23 @@ export type ResizeEvent = CustomEvent<{
  * @returns {void}
  */
 const triggerResize = (entry: ResizeObserverEntry): void => {
-
-  const { inlineSize: width, blockSize: height } = entry.borderBoxSize[0];
+  let width: number;
+  let height: number;
+  if (entry.borderBoxSize?.length > 0) {
+    width = entry.borderBoxSize[0].inlineSize;
+    height = entry.borderBoxSize[0].blockSize;
+  }
+  /* c8 ignore next 4 */
+  else {
```

Comment:
Using line count isn't really nice. Do we have any other way to accomplish this ?

Reply:
Other option is start and stop. How do you think?

/* c8 ignore start */
...
/* c8 ignore stop */

Reply:
Updated.

Reply:
It's a lot better.